lms/envs/common.py

```diff
@@ -2860,13 +2854,4 @@
 # to compete with the MOOC.
 CCX_MAX_STUDENTS_ALLOWED = 200
 
-STUDENT_RESPONSES_REPORT_SUPPORTED_TYPES = set([
-    'problem',
-    'submit-and-compare',
-    'freetextresponse',
-])
-
-# These types are children of children of units.
-TYPES_WITH_CHILD_PROBLEMS_LIST = [
-    'library_content',
-]
+from openedx.stanford.lms.envs.common import *
```

Comment:
Yeah. I intentionally placed it at the very bottom so that our values always take precedence over anything edX has defined.